### PR TITLE
Update subprojects to fix TUIO build, update package list.

### PR DESCRIPTION
**Pull request by rdumusc**

### CODE REVIEW

File: CMake/DisplayCluster.cmake [hunk 1]
```diff
@@ -13,8 +13,11 @@ set(DISPLAYCLUSTER_DEB_DEPENDS libavutil-dev libavformat-dev libavcodec-dev
   libopenmpi-dev openmpi-bin libswscale-dev libxmu-dev libpoppler-qt5-dev
   libboost-date-time-dev libboost-serialization-dev libboost-test-dev
   libboost-program-options-dev libboost-regex-dev libboost-system-dev
-  libboost-thread-dev libfcgi-dev)
-set(DISPLAYCLUSTER_PORT_DEPENDS ffmpeg boost poppler fcgi)
+  libboost-thread-dev libfcgi-dev qt5-default libqt5core5 libqt5declarative5
```
Comment (tribal-tec) [inline CMake/DisplayCluster.cmake]: qt5-default is not necessary imo and has potential side effects for other developments as it changes the system's default version used for any compilation.

File: CMake/DisplayCluster.cmake [hunk 2]
```diff
@@ -13,8 +13,11 @@ set(DISPLAYCLUSTER_DEB_DEPENDS libavutil-dev libavformat-dev libavcodec-dev
...
+  libqt5script5 libqt5scripttools5 libqt5svg5-dev libqt5webkit5-dev
+  libqt5xmlpatterns5-dev libqt5x11extras5-dev qtquick1-5-dev qtscript5-dev
+  libpoppler-qt5-dev)
+set(DISPLAYCLUSTER_PORT_DEPENDS ffmpeg boost poppler fcgi qt5-mac)
```
Comment (tribal-tec) [inline CMake/DisplayCluster.cmake]: you found a poppler qt5 mac ports?

Comment (rdumusc) [inline CMake/DisplayCluster.cmake]: no, poppler points to poppler-qt4-mac so looks like no pdf support on osx... Remove it or leave it in the hope that the port gets updated later?

File: CMake/DisplayCluster.cmake (hunk 1 above)
Comment (rdumusc) [inline CMake/DisplayCluster.cmake]: ok, I will check what it depends on and then remove it.

Comment (rdumusc) [inline CMake/DisplayCluster.cmake]: done, replaced by qtbase5-dev

File: CMake/DisplayCluster.cmake (hunk 2 above)
Comment (tribal-tec) [inline CMake/DisplayCluster.cmake]: remove for now

Comment (rdumusc) [inline CMake/DisplayCluster.cmake]: done

**Comment by rdumusc** [comment]
updated

**Pull request merged**